src/components/MainNav/MyProfile/MyProfile.js

```diff
@@ -47,6 +55,32 @@ export default class MyProfile extends Component {
     this.getProfileImage = this.getProfileImage.bind(this);
   }
 
+  componentDidMount() {
+    const links = this.props.modules.settings.map(setting => setting.links);
+
+    if (links && Array.isArray(links)) {
+      const linkItems = links[0].userDropdown;
+
+      linkItems.forEach((item, index) => {
+        this.isShownToUser().then((response) => {
+          const showToUser = response.status === 200;
+
+          if (showToUser) {
+            const itemCaption = item.caption;
+            const buttonId = 'clickable-chagepw' + index;
+            const newItem = (
+              <NavListItem id={buttonId} type="button" onClick={() => this.onChangePasswordHandler(item)}>
```

Comment:
Similarly, we can't wire an invocation of a change-password-handler into the user-menu! All we can do here -- and all we need to do -- is redirect to the URL that' been configured in the `items` array.

Reply:
changed